Cherry-pick: "Documenting the resubscribe_url token" to 7.1 (#620)
mautic/user-documentation: docs/channels/emails.rst

*************

Mautic has a built in means of allowing a Contact to unsubscribe from Email communication. You can insert various tokens into your Email to provide unsubscribe options at your desired location:

> - ``{unsubscribe_text}``: Inserts a sentence with a link instructing the Contact to click to unsubscribe.

Author: ```suggestion
- ``{unsubscribe_text}``: inserts a sentence with a link instructing the Contact to click to unsubscribe.
```

> - ``{unsubscribe_url}``: Inserts the URL to the preferences center when it's activated, or to the unsubscribe page if not.

Author: ```suggestion
- ``{unsubscribe_url}``: inserts the URL to the preferences center when it's activated, or to the unsubscribe page if not.
```

- ``{resubscribe_url}``: inserts the URL to the resubscribe page regardless of whether there's a preference centre in use. It resubscribes the Contact. Useful for double opt out Campaigns.

> - ``{dnc_url}``: Inserts the URL to unsubscribe from all marketing messages when the preference center is activated.

Author: ```suggestion
- ``{dnc_url}``: inserts the URL to unsubscribe from all Marketing Messages when you activate the preference center.
```
